universe/jupyterlab/marathon.json.mustache

```diff
@@ -14,38 +14,19 @@
   "container": {
     "type": "MESOS",
     "volumes": [
-
-      {{#service.service_account_secret}}
-      {
```

Comment:
We should retain this block service_account_secret block if it's present in the original config. Although I don't see why the secret name is hard coded. Try asking on the original repo, and tag vishnu2kmohan.